docs: deploy bytebase via rainbond (#622)
bytebase/bytebase.com: content/docs/get-started/install/deploy-to-rainbond.md

---
title: Deploy to Rainbond
---

This tutorial is for users who don't know Container and Kubernetes. 

If you don't have Container and Kubernetes knowledge, And want to install Bytebase in Kubernetes, you can use Rainbond to deploy bytebase. [Rainbond](http://www.rainbond.com) is cloud native application management platform built on Kubernetes, easy to use, no need to know Kubernetes knowledge, easily deploy applications in Kubernetes.

## Prerequisites

> You need to have a Rainbond. Please ref [Rainbond Quick Install](https://rainbond.com/docs/quick-start/quick-install).

Reviewer: ```suggestion
To install Rainbond, please refer to [Rainbond Quick Install](https://rainbond.com/docs/quick-start/quick-install).
```

## Deploy Bytebase

> After login in to Rainbond, click `Market` in the left menu, switch to open source app store, and search `Bytebase` in the search box, and click Install button.

Reviewer: ```suggestion
After logging in Rainbond, click `Market` in the left menu, switch to open source app store, and search `Bytebase` in the search box, and click the Install button.
```

> ![](/static/docs/get-started/install/rainbond-install.png)

Reviewer: Images need to be converted to webp format

> Fill in the installation information, and click Confirm button startup install.

Reviewer: ```suggestion
Fill in the following information, and click `Confirm` button to install.
```

* Team: select a team or create a new team
* Cluster: select a cluster
* Application: select an application or create a new application
* Version: select a version

> Moment later, Bytebase will be installed successfully, via the Access button to access Bytebase.

Reviewer: ```suggestion
After installation, Bytebase can be accessed via the `Access` button.
```

> > Via Rainbond installed Bytebase, external PostgreSQL is used by default. 

Reviewer: Let's used the hint block https://bytebase.com/docs/document-write-guide#hint

---
title: Deploy to Rainbond
---

> This tutorial is for users who don't know Container and Kubernetes. 
> 
> If you don't have Container and Kubernetes knowledge, And want to install Bytebase in Kubernetes, you can use Rainbond to deploy bytebase. [Rainbond](http://www.rainbond.com) is cloud native application management platform built on Kubernetes, easy to use, no need to know Kubernetes knowledge, easily deploy applications in Kubernetes.

Reviewer: ```suggestion

If you are unfamiliar with Kubernetes, and want to install Bytebase in Kubernetes, you can use Rainbond to deploy. [Rainbond](http://www.rainbond.com) is a cloud-native application management platform built on Kubernetes and simplifies the application deployment to Kubernetes.
```

## Prerequisites

To install Rainbond, please refer to [Rainbond Quick Install](https://rainbond.com/docs/quick-start/quick-install).

## Deploy Bytebase

After logging in Rainbond, click `Market` in the left menu, switch to open source app store, and search `Bytebase` in the search box, and click the `Install` button.

![](/static/docs/get-started/install/rainbond-install.webp)

Fill in the following information, and click `Confirm` button to install.

* Team: select a team or create a new team
* Cluster: select a cluster
* Application: select an application or create a new application
* Version: select a version

After installation, Bytebase can be accessed via the `Access` button.

<hint-block type="info">

> Via Rainbond installed Bytebase, external PostgreSQL is used by default. 
> 
> External URL for Bytebase is provided by Rainbond by default, You can go to the Bytebase component -> Environment configuration, Modify `EXTERNAL_URL` Environment Variables to customize the external URL.

Reviewer: ```suggestion
Rainbond uses [--external-url](/docs/get-started/install/external-url) to store the Bytebase metadata in the external PostgreSQL. You can go to the Bytebase component -> Environment configuration, Modify `EXTERNAL_URL` Environment Variables to customize the external URL.
```

Author: This is two paragraphs.

Rainbond uses [--pg](/docs/get-started/install/external-postgres) to store the Bytebase metadata in the external PostgreSQL.

Rainbond uses [--external-url](/docs/get-started/install/external-url) to provides external access to Bytebase. You can go to the Bytebase component -> Environment configuration, Modify `EXTERNAL_URL` Environment Variables to customize the external URL.

Reviewer: My bad. Got drunk

> Via Rainbond installed Bytebase, external PostgreSQL is used by default. 
> 
> External URL for Bytebase is provided by Rainbond by default, You can go to the Bytebase component -> Environment configuration, Modify `EXTERNAL_URL` Environment Variables to customize the external URL.

Reviewer: ```suggestion
Rainbond uses [--pg](/docs/get-started/install/external-postgres) to store the Bytebase metadata in the external PostgreSQL.

Rainbond uses [--external-url](/docs/get-started/install/external-url) to provide external access to Bytebase. You can go to the Bytebase component -> Environment configuration, and Modify EXTERNAL_URL Environment Variables to customize the external URL.
```

Reviewer: @zzzhangqi please see the new edit, but I have a question, do we provide the similar ENV variable for configuring external PG?

Author: Yes, provide. `POSTGRES_HOST` `POSTGRES_PORT` `POSTGRES_DB` `POSTGRES_USER` `POSTGRES_PASSWORD`
